### ISSUE TITLE
Eureka light client creation to avoid duplicate light clients between 2 chains

### ISSUE THREAD

**Issue by womensrights** [cosmos/ibc#1171]
With Eureka, the channel handshake is streamlined so that upon light client creation, a corresponding light client is registered to it, rather than a 4 step connection and 4 step channel handshake. 

In the case of IBC v1, multiple channels could be created upon a light client and connection which created some UX pains, particularly when using ICS-20 token transfer, as tokens sent across different channels were not fungible, despite being created on top of the same client and connection. With Eureka, there is a question whether this same problem could occur if multiple clients are created to connect the same 2 chains, even if they have the exact same security parameters.

This issue is to track that we should discuss whether this could be prevented so that there is just a single cannonical channel between two chains that want to use Eureka. It seems that this could be complicated to do without permissioned client creation.

**Comment by AdityaSripal** [comment]
I don't think this needs any changes to the spec, since this is not a required behavior of IBC implementations. Chains can choose to implement some form of non-duplication logic for convenience and if we wish we can discuss how to do so in an appendix of the spec
